### PR TITLE
Assignment 1 Question 2

### PR DESCRIPTION
**Pull request by UsmanNaveed1**
Added new articles on NLP

### CODE REVIEW

**Comment by UsmanNaveed1** [comment]
Added the following articles:
1. NLP: Extracting the main topics from your dataset using LDA in minutes
-Quick read about LDA and topic modelling

2. Over 150 of the Best Machine Learning, NLP, and Python Tutorials I’ve Found
- Super cool list of tutorials, includes some very nice tutorials on neural networks as well

3. Text Preprocessing in Python: Steps, Tools, and Examples
- Found this tutorial helpful while complete the first part of this assignment! So I think it would be a good resource for other folks as well 

4. Building a Simple Chatbot from Scratch in Python (using NLTK)
- Goes through how to build a very simple chatbot

5. 13 Deep Learning Frameworks for Natural Language Processing in Python
- Good list of deep learning frameworks, also provides a useful table for capabilities of each framework.

6. 7 types of Artificial Neural Networks for Natural Language Processing
-Not very detailed, but a good overview

Removed the following articles (I had mistakenly uploaded some deep learning resources so removing them in this request)
-Fast AI: Practical Deep Learning for Coders, v3
- Cutting Edge Deep Learning For Coders
- Introduction to Machine Learning for Coders
- Computational Linear Algebra for Coders

File: README.md [hunk 1]
```diff
@@ -95,7 +95,9 @@ A curated list of resources for learning about natural language processing, text
 
 ### General <a id="general-articles"></a>
 - [Natural Language Processing is Fun! How computers understand Human Language](https://medium.com/@ageitgey/natural-language-processing-is-fun-9a0bff37854e). 2018.
+- [NLP: Extracting the main topics from your dataset using LDA in minutes](https://towardsdatascience.com/nlp-extracting-the-main-topics-from-your-dataset-using-lda-in-minutes-21486f5aa925)
```
Comment (stepthom) [inline README.md]: This should go in the ConceptAnalysis/Topic Modeling section.

File: README.md [hunk 2]
```diff
@@ -95,7 +95,9 @@ A curated list of resources for learning about natural language processing, text
...
 - [WEF Live Campaign - Twitter fed Global News Topics & Sentiment Tracker](https://365.weflive.com/#!/r/worldwide/t/all-topics) - Live Jan 2019
+- [NLP — Building a Question Answering model](https://towardsdatascience.com/nlp-building-a-question-answering-model-ed0529a68c54). March 2018
```
Comment (stepthom) [inline README.md]: This should go in the Q&A Systems section.

File: README.md [hunk 3]
```diff
@@ -126,6 +128,7 @@ A curated list of resources for learning about natural language processing, text
 - [100 Must-Read NLP Papers](https://github.com/mhagiwara/100-nlp-papers)
 - [Crowdsourcing Ground Truth for Medical Relation Extraction](https://dl.acm.org/citation.cfm?id=3152889)
 - [Natural language based financial forecasting: a survey](https://sentic.net/natural-language-based-financial-forecasting.pdf)
+- [Over 150 of the Best Machine Learning, NLP, and Python Tutorials I’ve Found](https://medium.com/machine-learning-in-practice/over-150-of-the-best-machine-learning-nlp-and-python-tutorials-ive-found-ffce2939bd78)
```
Comment (stepthom) [inline README.md]: I would put this in the "Other Curated Lists" section.

**Comment by stepthom** [comment]
@UsmanNaveed1 Thank you! 👍

**Pull request merged**